src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(firstNumber, secondNumber) {
```

Comment:
```suggestion
    add(number) {
```
Try to make it work with one param